core/src/main/resources/hudson/security/LegacySecurityRealm/config.groovy

```diff
@@ -7,8 +7,8 @@ import jenkins.model.Jenkins
 def f = namespace(lib.FormTagLib)
 
 f.entry(title: _('Unprotected URLs')) {
-    p(class: "jenkins-form-description") {
-        _('blurb')
+    div(class: "jenkins-form-description") {
+       description(_('blurb'))
     }
```

Comment:
Seems to me like

```suggestion
    f.description(_('blurb'))
```

would be preferable, as that's how everything else uses `jenkins-form-description`, via https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/form/description.jelly

I don't think `<description>` is an HTML tag?

Reply:
Thanks @daniel-beck ! Your solution works perfectly, I just got confused by the jelly description you mention.